tests/components/channel_members_dropdown.test.jsx

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import ChannelMembersDropdown from 'components/channel_members_dropdown/channel_members_dropdown.jsx';
+
+describe('components/channel_invite_modal', () => {
+    const user = {
+        id: 'user-1',
+    };
+
+    const channel = {
+        create_at: 1508265709607,
+        creator_id: 'zaktnt8bpbgu8mb6ez9k64r7sa',
+        delete_at: 0,
+        display_name: 'testing',
+        extra_update_at: 1508265709628,
+        header: 'test',
+        id: 'owsyt8n43jfxjpzh9np93mx1wa',
+        last_post_at: 1508265709635,
+        name: 'testing',
+        purpose: 'test',
+        team_id: 'eatxocwc3bg9ffo9xyybnj4omr',
+        total_msg_count: 0,
+        type: 'O',
+        update_at: 1508265709607,
+    };
+
+    const baseProps = {
+        channel,
+        user,
+        channelMember: {
+            roles: 'channel_admin',
+            scheme_admin: 'system_admin',
+        },
+        isLicensed: true,
+        canChangeMemberRoles: false,
+        canRemoveMember: true,
+        actions: {
+            removeChannelMember: jest.fn().mockImplementation(() => {
+                const error = {
+                    message: 'Failed',
+                };
+
+                return Promise.resolve({error});
+            }),
+            getChannelStats: jest.fn(),
+            updateChannelMemberSchemeRoles: jest.fn(),
+        },
+    };
+
+    test('should match snapshot for channel_invite_modal', () => {
```

Comment:
Maybe `'should match snapshot for channel member remove button'`?

Reply:
Will change